<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Multi Cipher GUI</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>550</height>
     </rect>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QPushButton" name="pushButtonCaesar">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>150</y>
        <width>200</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>Caesar Cipher</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonRailway">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>250</y>
        <width>200</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>Railway Cipher</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelChoice">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>50</y>
        <width>250</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>Pilih Metode Cipher</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QLineEdit" name="lineEditInput">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>100</y>
        <width>300</width>
        <height>30</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Masukkan teks</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditShift">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>150</y>
        <width>300</width>
        <height>30</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Masukkan shift (angka)</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonEncrypt">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>200</y>
        <width>300</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>Encrypt</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelOutput">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>260</y>
        <width>300</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>Hasil akan tampil di sini</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonBack">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>320</y>
        <width>300</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <widget class="QLabel" name="labelRailway">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>200</y>
        <width>400</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>Railway Cipher (Coming Soon)</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonBackRailway">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>300</y>
        <width>200</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
